coderedcms/models/snippet_models.py

```diff
@@ -349,6 +349,71 @@ def __str__(self):
         return self.name
 
 
+@register_snippet
+class Accordion(ClusterableModel):
+    """Class for reusable content in a collapsible block."""
+    class Meta:
+        verbose_name = _('Accordion')
+        verbose_name_plural = _("Accordions")
+
+    name = models.CharField(
+        max_length=255,
+        verbose_name=_('Name'),
+    )
+
+    panels = (
+        [
+            MultiFieldPanel(
+                heading=_('accordion'),
+                children=[
+                    FieldPanel('name'),
+                ]
+            ),
+            InlinePanel('accordion_panels', label=_('Panels'))
+        ]
+    )
+
+    def __str__(self):
+        return self.name
+
+
+@register_snippet
+class AccordionPanel(Orderable, models.Model):
+    """A panel for a collapsible accordion"""
+
+    accordion = ParentalKey(
+        Accordion,
+        related_name='accordion_panels',
+        verbose_name=_('Accordion'),
+    )
+
+    name = models.CharField(
+        max_length=255,
+        verbose_name=_('Name'),
+    )
+
+    content = CoderedStreamField(HTML_STREAMBLOCKS, blank=True)
```

Comment:
Not sure why the unit tests _didn't_ throw a warning, but I'm pretty sure wagtail 3 now has a required argument `use_json_field=True`

https://docs.wagtail.org/en/stable/releases/3.0.html#use-json-field-argument-added-to-streamfield